html-fundamentals/content/nutrition.html

```diff
@@ -21,10 +24,64 @@ <h2>Nutrition Information</h2>
             To that end, here is the nutritional information for our products.
         </p>
 
+        <table>
+            <caption>Wired Brain Nutrition Information</caption>
+            <thead>
+                <tr>
+                    <th rowspan="2">Drink</th><!--cell uses two rows-->
+                    <th></th>
+                    <th></th>
+                    <th></th>
+                    <th></th>
+                    <th colspan="2">Carbohydrates</th><!--cell uses two columns-->
```

Comment:
this why the attribute starts with row and col, no need for comments

Reply:
true, but still would like to keep them for future ref, Im an old man

Reply:
fair enough old man